tests/unit/composables/LetoModelizerApi.test.js

```diff
@@ -99,4 +100,26 @@ describe('User Authentication', () => {
       expect(res.results).toEqual(roles);
     });
   });
+
+  describe('Test function: manageError', () => {
+    it('should throw an error on not 503', async () => {
```

Comment:
should throw an error if status is not 503

Reply:
done